playlists/word.yaml

```diff
@@ -16,6 +16,15 @@
   group: Basics
   api_set:
     WordApi: 1.1
+- id: word-01-basics-multiple-property-set-
```

Comment:
Ricky, could you please move this out of `Basics` and... not quite sure where :-)?  Maybe we need a "general patterns" group.  But Basics is definitely the wrong place, and causes clutter.

Ditto for the Excel one.